### ISSUE TITLE
New design issues

### ISSUE THREAD

**Issue by vvo** [algolia/instantsearch.js#1472]
@LukyVj This is an issue to track the glitches introduced via the awesome new docs design of react-instantsearch. Will be easier for you to track as a single issue. Anyone can jump in and add an item, then you check them
- [x] examples do not have an header anymore, the new header should be ported. Or at least make the examples aware of the react-instantsearch context so the user knows how to go back. We could also use iframes here, that would be OK :)
- [x] there's no link on the Algolia timer in hero
- [x] there's a bug on the community dropdown on small screen
- [x] the awesome logo and animation is too slow to show, easy to miss it. I would show it right away so the experience is not too surprising if you started to read the page already
- [x] Can you beautify the new introduction pages for guides, components, connectors and examples (don't hesitate to ping me if you have any template related question, @bobylito)
- [x] there's no Algolia logo in header on small screen
----
- [ ] ~when resizing the doc window in half, there's no more menu to switch between widgets/guides/etc. Also the title/logo in the header kinda disappear. Then when resizing the window to fit all the screen, the right menu is being displayed on the header.~ [ **It's not a bug, it's a feature** ]  
- [ ] code snippet are not highlighted in the  guide section (only css/html remaining for now) [ @bobylito  ] 
- [x] mobile hamburger menu works once but if you resize window back and forth it will stop working
- [x] intro page for api (connectors & widgets) doesn't expand the body even if enough item in the sidebar. 
- [x] material-ui, media and tourism examples are still broken. Seems to be a fixed header issue. Also for material-ui, it seems that some styles are missing
- [x] a too large value inside a prop table broke the layout (see HOC/HierarchicalMenu.html)

**Comment by LukyVj** [comment]
Hey @vvo for the task : 
>  there's no Algolia logo in header on small screen

If you talk about this : 
![](https://puu.sh/s1I1v/717dbde70a.png)

It's made on purpose

**Comment by LukyVj** [comment]
@vvo you mean "not animate it" ? 

> the awesome logo and animation is too slow to show, easy to miss it. I would show it right away so the experience is not too surprising if you started to read the page already

**Comment by vvo** [comment]
> It's made on purpose

Works for me, I just wanted to point that there's then no way to go to Algolia.com on mobile easily versus desktop.

**Comment by vvo** [comment]
> @vvo you mean "not animate it" ?

No, just that it seems as if there was some (long) timer before we start showing it. Sometime I already started browsing the page/reading and the logo animation starts (but it's "too late", I won't see it)

**Comment by vvo** [comment]
small screen glitch:

![2016-10-31-161731_1410x840_scrot](https://cloud.githubusercontent.com/assets/123822/19859464/a06ad756-9f85-11e6-93b1-f7f4011f6e27.png)

**Comment by vvo** [comment]
small screen glitch:

![2016-10-31-161731_1410x840_scrot](https://cloud.githubusercontent.com/assets/123822/19859464/a06ad756-9f85-11e6-93b1-f7f4011f6e27.png)

**Comment by LukyVj** [comment]
So, for the animation task : 

> The awesome logo and animation is too slow to show, easy to miss it [...]

I decided to add two variables to handle that. That way, you will be able to setup the desired animation length and delay, I already reduced the delay before the animation starts, but still, if you want to test, you can modify the value  of the variables : 
```
$hero-logo-anim-length: 5s;
$hero-logo-anim-delay: 0.3s;
```

**Comment by mthuret** [comment]
Just looking at the doc after adding all the widgets/connectors jsdoc, and the footer seems to take a lot of space making the sidebar scrolling. 

![capture d ecran 2016-11-03 a 10 33 15](https://cloud.githubusercontent.com/assets/6885378/19961096/fd75d87e-a1b0-11e6-8085-df5b7754e7c0.png)

Do you have the same issue?

**Comment by vvo** [comment]
From my POV the current footer:
- is taking too much visual space, we are used to it but I would not be surprised people try to make sense of this area while they should not have to
- is not in line with the new header

I am sure we will be able to solve that in the coming weeks once @LukyVj can work again with us. In the meantime any input from @LukyVj @AntoinePlu @Shipow is welcomed!

**Comment by vvo** [comment]
@mthuret Could you open a new issue instead of reopening this one? So that it's easier to understand.

**Comment by LukyVj** [comment]
Huh, the footer is the community footer :/ We designed it and it's present on each and every community website, if we have to change it, we would have to change it everywhere, so that wont happen right now :/ Sorry. 

We'll do our best

**Comment by mthuret** [comment]
Little bit more precision here: it only appears on the introduction page (if you choose an item in the sidebar, you'll get a big body). So I assume that's an issue with the template used for intro.

**Comment by LukyVj** [comment]
Oh, I see what you mean, keep this thread open, I'll fix that on monday :) \o/ 

Thanks for your reactivity!

**Comment by mthuret** [comment]
There's also some examples that still have issues. I updated this issue with the problem. 
I'll try to understand and fix what's wrong in the meantime :)

**Comment by LukyVj** [comment]
Really ? I checked them all yesterday, I didn't spotted anything wrong, but maybe I missed them. All I know is that I fixed the community header & I fixed the airbnb demo. If you need urgent help, call me \o/

**Comment by vvo** [comment]
As for the footer styling, I understand it's currently common to the whole community. I hope we can still make it evolve as we updated the header, the hero sizing. "At some point" :)

**Comment by LukyVj** [comment]
Yup, it's in the `|`

**Comment by mthuret** [comment]
Just to let you know that I'm currently fixing the examples (material-ui is the only one that's left). Examples break because of the usage of bootstrap in both the doc website and examples).
We we'll rewrite the examples soon so hopefully we will manage to reproduce those kind of issue but for now I just try to redefine bootstrap style for the examples. 
So keep in mind that if you modify a bootstrap style for the website, their's a chance it will break the examples.

**Comment by mthuret** [comment]
![image](https://cloud.githubusercontent.com/assets/6885378/19978584/bbba3e7e-a1f7-11e6-8ed4-d740896e024d.png)

But maybe the solution here is to shorten the title :)

**Comment by vvo** [comment]
> But maybe the solution here is to shorten the title :)

Yes. "3rd party libraries" is enough

**Comment by LukyVj** [comment]
> when resizing the doc window in half, there's no more menu to switch between widgets/guides/etc. Also the title/logo in the header kinda disappear. Then when resizing the window to fit all the screen, the right menu is being displayed on the header.

Can't reproduce it /o\
Help ?

**Comment by mthuret** [comment]
seems that the hamburger menu on the right is back :)
Still if you're resizing the window if half on small screen (like a macbook), there no more sidebar for guides/api... and nothing to replace it so that can be tricky to navigate inside.

**Comment by LukyVj** [comment]
For the : 
>  code snippet are not highlighted in the guide section (only css/html remaining for now)

I think @bobylito got the answer :)

**Comment by LukyVj** [comment]
@vvo @mthuret Who created that issue ? 
>  a too large value inside a prop table broke the layout (see HOC/HierarchicalMenu.html)

**Comment by bobylito** [comment]
@vvo @LukyVj It is highlighted on my current branch 😕 Probably something was removed in the meantime. Can you check @LukyVj ?

**Comment by LukyVj** [comment]
Yup, where should I check @bobylito ?

**Comment by mthuret** [comment]
@bobylito @LukyVj : js is highlighted, but not the css/html example (guide/styling)

**Comment by vvo** [comment]
Let's do another round of design issues
